lib/Pipes/PipelineManager.cpp

```diff
@@ -47,6 +47,8 @@ class LoadModelPipePass {
   void registerPasses(llvm::legacy::PassManager &Manager) {
     Manager.add(new LoadModelWrapperPass(Wrapper));
   }
+
+  void print(llvm::raw_ostream &OS) const { OS << ""; }
```

Comment:
Why empty?

Reply:
this one is actually legit, the load model pass is added by revng opt by default, so we don't need to add it ourself

Reply:
this one is actually legit, the load model pass is added by revng opt by default, so we don't need to add it ourself

Reply:
i guess i can leave a comment